### PR TITLE
use webv 2.0-beta1 to help test

### PR DESCRIPTION
**Pull request by bartr**

### CODE REVIEW

**Comment by kforeverisback** [comment]
Verified Readme links
Created Codespaces and verified
`make check` works as document specified

File: .devcontainer/post-install.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
-# install WebV
+# install WebV (using the beta for testing)
 # we can't install this in the base image
-dotnet tool install -g webvalidate
+dotnet tool install -g --version 2.0.0-beta1 webvalidate
```
Comment (gortegaMS) [inline .devcontainer/post-install.sh:5]: Is version 2.0.0-beta1 already available in git hub packages container?

**Review by gortegaMS — APPROVED** [review]
Looks good to me, just have a quick question.  Please see my comments.

File: .devcontainer/post-install.sh (hunk 1 above)
Comment (bartr) [reply .devcontainer/post-install.sh:5]: Yes, it's on nuget

**Pull request merged**